packages/utils/src/object.js

```diff
@@ -179,3 +179,8 @@ export const mapValues = (object, mapping, { defaultToExistingValues = false } =
 
   return result;
 };
+
+export const stripFields = (object, fieldsToExclude) =>
+  Object.entries(object)
+    .filter(([key]) => !fieldsToExclude.includes(key))
+    .reduce((strippedObject, [key, value]) => ({ ...strippedObject, [key]: value }), {});
```

Comment:
The above is 100% fine but just thought you may appreciate the power of `Object.fromEntries`:

```suggestion
  Object.fromEntries(Object.entries(object).filter(([key]) => !fieldsToExclude.includes(key)));
```

Reply:
Cool!